docs/python/jupyter-support.md

```diff
@@ -11,134 +11,136 @@ MetaSocialImage: images/tutorial/social.png
 
-```python
-#%%
-msg = "Hello World"
-print(msg)
+![Blank Jupyter Notebook](images/jupyter/native-code-cells-01.png)
+
+If you have an existing Jupyter Notebook, you can open it in the Notebook Editor by double clicking on the file and opening with Visual Studio Code, through the Visual Studio Code, or using the Command Palette **Python: Open in Notebook Editor** command.
+
+## Work with code cells in the Notebook Editor
+
+The Notebook Editor makes it easy to create, edit, and run code cells within your Jupyter Notebook.
+
+### Create a code cell
+
+By default, a blank Notebook will have an empty code cell for you to start with and an existing Notebook will place one at the bottom. Simply add your code to the empty code cell to get started.
 
-#%%
-msg = "Hello again"
+```python
+msg = "Hello world"
 print(msg)
 ```
 
-> **Note**: Make sure to save the code shown above in a file with a .py extension.
+![Simple Jupyter code cell](images/jupyter/native-code-cells-02.png)
 
-When the Python extension detects a code cell, it adds **Run Cell** and **Debug Cell** CodeLens adornments. The first cell also includes **Run Below** and all subsequent cells include **Run Above** :
+### Add additional code cells
 
-![Jupyter adornments for code cells in the VS Code editor](images/jupyter/code-cells-01.png)
+Code cells can be added to a Notebook using the main toolbar, a code cell's vertical toolbar, the add code cell icon at the bottom of the notebook, the add code cell icon at the top of the notebook (visible with hover), and through keyboard commands.
 
-> **Note:** By default, **Debug Cell** just steps into user code. If you want to step into non-user code, you need to uncheck **Data Science: Debug Just My Code** in the Python extension settings (`kb(workbench.action.openSettings)`).
+![Add code cells](images/jupyter/native-code-cells-07.png)
 
-**Run Cell** applies to only the one code cell. **Run Below**, which appears on the first cell, runs all the code in the file. **Run Above** applies to all the code cells up to, but not including, the cell with the adornment. You would use **Run Above**, for example, to initialize the state of the runtime environment before running that specific cell.
+ Using the plus icon in the main toolbar will add a new cell directly below the currently selected cell. Using the add cell icons at the top and bottom of the Jupyter Notebook, will add a code cell at the top and bottom respectively. And using the add icon in the code cell's toolbar, will add a new code cell directly below it.
 
-Selecting a command starts Jupyter (if necessary, which might take a minute), then runs the appropriate cell(s) in the **Python Interactive** window:
+When a code cell is selected, the `kbstyle(A)` key can be used to add a cell above and the `kbstyle(B)` can be used to add a cell below.
```

Comment:
Not selected persay, but more in "command mode". Since there are 4 states to a cell, unselected, hovered, cell focus (you cant edit the code cell), content focus (you can edit the code cell). Command mode can be achieved by selecting outside the cell, or pressing "ESC" when you are editing code.